test/unified_memory_allocation/common/main.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#include <gtest/gtest.h>
+
+int main(int argc, char **argv) {
```

Comment:
Adding a custom `main()` function that doesn't do anything special is not necessary.

Reply:
What do you mean? This is necessary for gtest I belive.

Reply:
Gtest will provide it's main function and run all tests by default. See new tests added here, ie: https://github.com/oneapi-src/unified-runtime/pull/154/commits/c65c4417dfc1845bd9796f463d9589d0bf2c77ff

Reply:
I see, thanks. Removed.